servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -292,11 +292,55 @@ public String toString() {
     return sb.toString();
   }
 
+  /**
+   * Converts the entry to an {@code InodeFile}.
+   *
+   * @return the {@code InodeFile} representation
+   */
+  public static InodeFile fromJournalEntry(InodeFileEntry entry) {
+    InodeFile inode =
+        new InodeFile.Builder()
+            .setName(entry.getName())
+            .setBlockContainerId(BlockId.getContainerId(entry.getId()))
+            .setBlockSizeBytes(entry.getBlockSizeBytes())
+            .setCacheable(entry.getCacheable())
+            .setCreationTimeMs(entry.getCreationTimeMs())
+            .setLastModificationTimeMs(entry.getLastModificationTimeMs())
+            .setParentId(entry.getParentId())
+            .setPersisted(entry.getPersisted())
+            .setPinned(entry.getPinned())
+            .setTTL(entry.getTtl())
```

Comment:
I don't want to add more complexity to this PR, so I'll do the rename in a followup PR or let a new contributor do it. I've created https://tachyon.atlassian.net/browse/TACHYON-1335, which I'll probably unassign once this PR merges.